modules/announcements/assets/js/components/AnnouncementBody.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+
+export default function AnnouncementBody( { announcement } ) {
+	const { title, description, media } = announcement;
+
+	return (
+		<div className="announcement-body-container">
+			<div className={ `announcement-body-media announcement-body-${ media.type }` }>
+				{ 'image' === media.type ? (
+					<img src={ media.src } alt="Announcement" />
+				) : (
+					<video controls width="100%">
+						<source src={ media.src } type="video/mp4" />
+						Sorry, your browser doesn&apos;t support videos. </video>
```

Comment:
At the moment we don't need video so please remove it.

Btw- For next time we can 
1. Create component for each type ( image and video )
2. We always need to use string's localization with the textdomain.